requirements/requirements_git.txt

```diff
@@ -5,4 +5,4 @@ git+https://github.com/ansible/ansible-runner.git@devel#egg=ansible-runner
 # specifically need https://github.com/robgolding/django-radius/pull/27
 git+https://github.com/ansible/django-radius.git@develop#egg=django-radius
 git+https://github.com/ansible/python3-saml.git@devel#egg=python3-saml
-git+https://github.com/ansible/django-ansible-base.git@devel#egg=django-ansible-base[filtering]
+django-ansible-base @ git+https://github.com/john-westcott-iv/django-ansible-base@app_split#egg=django-ansible-base[rest_filterscd]
```

Comment:
what is the prefix part `django-ansible-base @`?

also is `[rest_filterscd]` correct?

Reply:
the name was change

https://github.com/ansible/django-ansible-base/pull/76/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711

But it looks like it was renamed to `rest_filters`

Reply:
fixed the typo